src/core/pyproxy.c

```diff
@@ -11,6 +11,9 @@
 #include "pyproxy.h"
 #include "python2js.h"
 
+#define Py_ENTER()
```

Comment:
Yeah. Let's wait to see if it makes sense over there before merging this though. There is some trouble in `iter_helper`, but it seems to otherwise work okay.